### DISCUSSION TITLE
新开umi4项目，没有报错信息，该如何配置

### DISCUSSION THREAD

**Discussion by Luxiaoshi** [umijs/umi#9314] [Q&A]
<img width="404" alt="image" src="https://user-images.githubusercontent.com/8713879/190038755-f78c46ab-609e-4743-b9d1-9672d1a17997.png">
尽管代码里有这样的提示错误，但是页面依旧可以打开，并且没有错误信息和提示，想请教如何配置

**Comment by xiaorong61** [comment]
forkTSChecker: {},
